editor.planx.uk/src/@planx/components/Pay/Public/FeeBreakdown/utils.ts

```diff
@@ -32,11 +64,31 @@ export const createPassportSchema = () => {
     .union([questionSchema, setValueSchema])
     .transform(toNumber);
 
+  const amountsSchema = z.object({
+    "application.fee.calculated": feeSchema.optional().default(0),
+    "application.fee.payable": feeSchema,
+    "application.fee.payable.vat": feeSchema.optional().default(0),
+  });
+
+  const reductionsSchema = z
+    .record(
+      z.string(),
+      z.tuple([z.enum(["true", "false"])]).transform(toBoolean)
+    )
+    .optional();
+
+  const exemptionsSchema = z
+    .record(
+      z.string(),
+      z.tuple([z.enum(["true", "false"])]).transform(toBoolean)
+    )
+    .optional();
+
   const schema = z
     .object({
-      "application.fee.calculated": feeSchema.optional().default(0),
-      "application.fee.payable": feeSchema,
-      "application.fee.payable.vat": feeSchema.optional().default(0),
+      amount: amountsSchema,
+      reductions: reductionsSchema,
+      exemptions: exemptionsSchema,
```

Comment:
Breaking these down into their own schemas solved the known / unknown keys issue described here - https://github.com/theopensystemslab/planx-new/pull/4022/files#r1865505036

I was hitting some issues using `z.intersection()` or `z.and()` to combine these into a flat schema.